FreedomStudio/E31FPGA/coreplexip_welcome/.project

```diff
@@ -114,6 +119,26 @@
 			<type>2</type>
 			<locationURI>virtual:/virtual</locationURI>
 		</link>
+		<link>
+			<name>bsp/libwrap/.DS_Store</name>
+			<type>1</type>
+			<locationURI>PARENT-3-PROJECT_LOC/bsp/libwrap/.DS_Store</locationURI>
```

Comment:
Are these links to `.DS_Store` necessay?

Reply:
Good catch, I will remove.